lite/tests/classification/conftest.py

```diff
@@ -173,35 +171,24 @@ def classifications_image_example() -> list[Classification]:
         Classification(
             uid="uid5",
             groundtruths=[
-                ("k4", "v4"),
-                ("k5", "v5"),
```

Comment:
I cut the "k5" and "k3" entries out of this data, rather than replicating our tests 3x everywhere this data is used, because these keys didn't seem worth repeating as separate, unique tests: they both just test for a false negative / false positive combo, which is already tested in "uid5" by gt=['v4'] and pd=['v1', 'v8']